Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -359,7 +359,8 @@ void TBWidget::AddChildRelative(TBWidget *child, WIDGET_Z_REL z, TBWidget *refer
 
 void TBWidget::RemoveChild(TBWidget *child, WIDGET_INVOKE_INFO info)
 {
-    assert(child->m_parent);
+    if (!child->m_parent)
+        return;
 
```

Comment:
I think this is a good change, as long as we are tripping the assert by explicitly calling RemoveChild.  If we're hitting the assert as a side effect of something, that should be looked into